.travis.yml

```diff
@@ -22,10 +30,7 @@ before_install:
   - mysql -e 'CREATE DATABASE IF NOT EXISTS test;'
   - sudo mysql -e "use mysql; update user set authentication_string=PASSWORD('root') where User='root'; update user set plugin='mysql_native_password';FLUSH PRIVILEGES;"
   - sudo apt-get -y install libvirt-dev
-
-install:
-  - pip install -r requirements.txt
-  - pip install codecov
+  - pip install pipenv
```

Comment:
Could you please move this above Line#14? If we have two jobs, we need a bit more separation. E.g. If you look at the job which uses `pip install requirements.txt` : [here](https://travis-ci.org/saurabhshri/ccextractor-web/jobs/447100525/config), you'll see it's also installing `pipenv`. There's no need to install it in this case.